events/wmt13.md

```diff
@@ -0,0 +1,158 @@
+| 11:00 – 12:30 | **Shared Task: Evaluation** |
+| | [***Ranking Translations using Error Analysis and Quality Estimation***](https://www.aclanthology.org/W13-2251.pdf) <br>Mark Fishel |
+| | [***Are ACT’s Scores Increasing with Better Translation Quality?***](https://www.aclanthology.org/W13-2252.pdf) <br>Najeh Hajlaoui |
+| | [***A Description of Tunable Machine Translation Evaluation Systems in WMT13 Metrics Task***](https://www.aclanthology.org/W13-2253.pdf) <br>Aaron Li-Feng Han, Derek F. Wong, Lidia S. Chao, Yi Lu, Liangye He, Yiming Wang, Jiaji Zhou |
+| | [***MEANT at WMT 2013: A Tunable, Accurate yet Inexpensive Semantic Frame Based MT Evaluation Metric***](https://www.aclanthology.org/W13-2254.pdf) <br>Chi-kiu Lo, Dekai Wu |
+| | [***An Approach Using Style Classification Features for Quality Estimation***](https://www.aclanthology.org/W13-2255.pdf) <br>Erwan Moreau, Raphael Rubino |
+| | [***DCU Participation in WMT2013 Metrics Task***](https://www.aclanthology.org/W13-2256.pdf) <br>Xiaofeng Wu, Hui Yu, Qun Liu |
+| 12:30 – 14:00	| 🍴 |
+| 14:00 – 14:20 | **Session 8: Reordering and Hierarchical Models** <br>[***Efficient Solutions for Word Reordering in German-English Phrase-Based Statistical Machine Translation***](https://www.aclanthology.org/W13-2257.pdf) <br>Arianna Bisazza, Marcello Federico |
+| 14:20 – 14:40 | [***A Phrase Orientation Model for Hierarchical Machine Translation***](https://www.aclanthology.org/W13-2258.pdf) <br>Matthias Huck, Joern Wuebker, Felix Rietig, Hermann Ney |
+| 14:40 – 15:00	| [***A Dependency-Constrained Hierarchical Model with Moses***](https://www.aclanthology.org/W13-2259.pdf) <br>Yvette Graham |
+| 15:00 – 15:20	| [***Investigations in Exact Inference for Hierarchical Translation***](https://www.aclanthology.org/W13-2260.pdf) <br>Wilker Aziz, Marc Dymetman, Sriram Venkatapathy |
+| 15:20 – 16:00	| ☕️ |
+| 16:00 – 16:20	| **Session 9: Alignment and Word Translation Models** <br>[***Evaluating (and Improving) Sentence Alignment under Noisy Conditions***](https://www.aclanthology.org/W13-2261.pdf) <br>Omar Zaidan, Vishal Chowdhary |
+| 16:20 – 16:40	| [***Multi-Rate HMMs for Word Alignment***](https://www.aclanthology.org/W13-2262.pdf) <br>Elif Eyigöz, Daniel Gildea, Kemal Oflazer |
+| 16:40 – 17:00	| [***Hidden Markov Tree Model for Word Alignment***](https://www.aclanthology.org/W13-2263.pdf) <br>Shuhei Kondo, Kevin Duh, Yuji Matsumoto |
+| 17:00 – 17:20	| [***An MT Error-Driven Discriminative Word Lexicon using Sentence Structure Features***](https://www.aclanthology.org/W13-2264.pdf) <br>Jan Niehues, Alex Waibel |
+
+## Results
+
+Full results of the shared tasks: [*Findings of the 2013 Workshop on Statistical Machine Translation*](https://www.statmt.org/wmt13/pdf/WMT01.pdf)
+
+### News translation
+
+The results are determined through human evaluation featuring [relative ranking](wmt.md#relative-ranking).
+The winner systems are listed according to their score.
```

Comment:
I think this can always be removed.

Could add this everywhere:
` ... | Score ↓ |`

Reply:
> I think this can always be removed.
> 
> Could add this everywhere: ` ... | Score ↓ |`

What if we just remove the sentence? Do we need the '↓' sign next to the `| Score  |` if the scores are ordered under the same column?